docs/content/en/open_source/ldap-authentication.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Authentication via LDAP"
+description: "Authenticate users using LDAP"
+draft: false
+weight: 4
+---
+
+## LDAP Authentication
+
+Out of the box Defect Dojo does not support LDAP authentication.
+
+*However*, since Defect Dojo is built using Django, it isn't too difficult to add support for LDAP.
+So long as you don't mind building your own Docker images...
+
+We will need to modify a grand total of 4-5 files, depending on how you want to pass Dojo your LDAP secrets.
+
+ - Dockerfile.django-*
+ - Dockerfile.nginx-*
+ - requirements.txt
+ - settings.dist.py
+ - docker-compose.yml *(Optional)*
+
+
+#### Dockerfile modifications
+
+In both Dockerfile.django and Dockerfile.nginx, you want to add the following lines to the apt-get install layers:
+
+```bash
+libldap2-dev \
+libsasl2-dev \
+ldap-utils \
+```
+
+
+#### requirements.txt
+
+Please check for the latest version of these requirements at the time of implementation on pypi.org and use those if you can.
+
+- [https://pypi.org/project/python-ldap/](python-ldap)
+- [https://pypi.org/project/django-auth-ldap/](django-auth-ldap)
+
+Otherwise add the following to requirements.txt:
+
+```python
+python-ldap==3.4.2
+django-auth-ldap==4.1.0
+```
+
+
+#### settings.dist.py
+
+Find the settings file (hint: `/dojo/settings/settings.dist.py`) and add the following:
```

Comment:
We generally discourage editing `settings.dist.py` now... I think `local_settings.py` would be the appropriate file to reference here? CC @Maffooch

Reply:
Agreed. Here is how I would word it, but feel free to wordsmith it 😄 
```suggestion
Find the settings file (hint: check in `/dojo/settings/settings.py` for instructions for how to use `/dojo/settings/local_settings.py`, if the file does not already exist) and add the following:
```